Sprint-1/explore/chrome.md

```diff
@@ -16,3 +16,7 @@ Now try invoking the function `prompt` with a string input of `"What is your nam
 
 What effect does calling the `prompt` function have?
 What is the return value of `prompt`?
+
+
+**The *"input"* is executed as seen in the screenshot above. The *string value* of the input is returned:**
+![alt text](image.png)
```

Comment:
If you were writing a program that uses `prompt()` to ask for an input value, how can your program tell if the user clicked "OK" or "Cancel"?

Reply:
- If the user clicks "OK" without entering anything, `prompt()` returns an empty string `""`.
- If the user clicks "OK" after entering a value, `prompt()` returns that value as a string.
- If the user clicks "Cancel", `prompt()` returns `null`